contracts/interfaces/IPoolDataProvider.sol

```diff
@@ -222,10 +230,9 @@ interface IPoolDataProvider {
    * @param asset The address of the underlying asset of the reserve
    * @return irStrategyAddress The address of the Interest Rate strategy
    */
-  function getInterestRateStrategyAddress(address asset)
-    external
-    view
-    returns (address irStrategyAddress);
+  function getInterestRateStrategyAddress(
```

Comment:
Well i would approve of this. However it seems it was developed by anger or just plain selfishness but I could be wrong I'll take a chance.